README.md

```diff
@@ -1,24 +1,21 @@
-# Docker and FreeScribe Installation Script
+# ClincianFOCUS Toolbox Installer
 
-This script helps automate the process of setting up Docker containers and installing the FreeScribe client, along with handling different model options (local or Hugging Face).
+This project automates the setup of Docker containers and the installation of the FreeScribe client.
 
 ## Prerequisites
 
-Before running the script, ensure you have the following installed on your system:
+Ensure you have the following installed on your system before running the script:
 
-- **Docker**: If Docker is not installed, the script will prompt you to install it. You can follow the installation guide [here](https://docs.docker.com/install/).
+- **Docker**: If Docker is not installed, the script will prompt you to install it. Follow the installation guide [here](https://docs.docker.com/install/).
 - **Docker Compose**: If Docker Compose is not installed, the script will also prompt you to install it. Follow the installation guide [here](https://docs.docker.com/compose/install/).
 
-- Only supports windows.
+- This script is designed to run on Windows environments only.
 
 ## Running the Script
 
-1. Ensure you have Docker and Docker Compose installed.
-2. Download the source code found here. Extract the files
-3. Run the script via the command line or double-click the `.bat` file.
-4. Follow the prompts to complete the installation.
+- Download the exe from releases.
```

Comment:
Too bad what Sourcery gave was not enough to just commit, but yes, this would be fair to add a bit more clarity.   Next time you are in.   Issue ticket made.